composables/massmint/parsers/parseTxt.ts

```diff
@@ -91,6 +94,7 @@ const updateEntries = (entries, block) => {
       file: entry.file,
       name: entry.name,
       description: entry.description,
+      tags: entry.attributes,
```

Comment:
`tags` and `attribute` let's keep one unless there is reason

Reply:
I guess the `attribute` is more user-friendly and easy to understand, so I put `attributes` file in the template file. However, when we call the polkdot API, we must use `tags` in the nft instead.

Reply:
Fair enough, but I would still keep one naming across the app **just for the sake of consistency** in this case attributes, since its already used in other places

See https://github.com/kodadot/nft-gallery/blob/main/components/common/EditNftModal.vue

And tags which is specific of SUB can be used inside transaction token mint logic

Let me know if it makes sense

Reply:
Alright, I have updated it.